com.streetwriters.notesnook.yml

```diff
@@ -0,0 +1,75 @@
+app-id: com.streetwriters.notesnook
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "22.08"
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node16
+command: start-notesnook
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=system-bus
```

Comment:
You need to figure which D-Bus name you need. This is not allowed.

Reply:
Fixed